server/pulp/server/async/scheduler.py

```diff
@@ -425,4 +473,5 @@ def apply_async(self, entry, publisher=None, **kwargs):
             has_failure = bool(entry._scheduled_call.consecutive_failures)
             self._failure_watcher.add(result.id, entry.name, has_failure)
             _logger.debug(_('watching task %s') % {'id': result.id})
+            _logger.info(_('watching task %s') % {'id': result.id})
```

Comment:
This should be logged at either `debug` or `info` but not both. I'll suggest doing it at `info` so keep this line and delete the one above it.